lib/ts/recipe/thirdpartypasswordless/types.ts

```diff
@@ -63,7 +63,10 @@ export type User = (
 export type TypeInput = (
     | {
           contactMethod: "PHONE";
-          validatePhoneNumber?: (phoneNumber: string) => Promise<string | undefined> | string | undefined;
+          validatePhoneNumber?: (
+              phoneNumber: string,
+              tenantId: string
+          ) => Promise<string | undefined> | string | undefined;
 
           // Override to use custom template/contact method
           /**
```

Comment:
createAndSendCustomTextMessage needs to be removed

Reply:
Even the email version

Reply:
Done